templates/dashboard/domain_detail/dns.html

```diff
@@ -285,24 +288,28 @@ <h1 class="h2">{{ custom_domain.domain }}</h1>
         {% if not dkim_ok %}
 
           <div class="text-danger mt-4">
-            Your DNS is not correctly set.
-            {% if dkim_errors %}
-
-              The CNAME record we obtain for
-              <em>dkim._domainkey.{{ custom_domain.domain }}</em> is:
-              <div class="mb-3 p-3 dns-record">
-                {% for r in dkim_errors %}
+            <p>
+              Your DNS is not correctly set.
+            </p>
+            <ul>
+              {% for custom_record, retrieved_cname in dkim_errors.items() %}
 
-                  {{ r }}
-                  <br />
-                {% endfor %}
-              </div>
-            {% endif %}
+                <li>
+                  The CNAME record we obtain for <em>{{ custom_record }}</em> is {{ retrieved_cname }}
+                </li>
+              {% endfor %}
+            </ul>
             {% if custom_domain.dkim_verified %}
 
               Without DKIM setup, emails you sent from your alias might end up in Spam/Junk folder.
             {% endif %}
           </div>
+          {% if custom_domain.dkim_verified %}
```

Comment:
this is shown even if the domain has all the DKIM records correctly set up right?

Reply:
It should not because it's under `if not dkim_ok`. So it should only show up if there are errors when validating dkim and it's still verified.